payments/swish/seamless-view.md

```diff
@@ -18,13 +18,19 @@ sidebar:
 
 {% include alert-development-section.md %}
 
-> Swish is a one-phase payment instrument supported by the major Swedish banks. In
-  the direct e-commerce scenario, Swedbank Pay receives the Swish registered
-  mobile number directly from the merchant UI. Swedbank Pay performs a payment
-  that the payer confirms using her Swish mobile app.
+{% include jumbotron.html body="Swish is a one-phase payment instrument
+supported by the major Swedish banks. In the direct e-commerce scenario,
```

Comment:
Isn't *Seamless View* and *Direct* two different scenarios / integration variants?